es6/docxtemplater.js

```diff
@@ -46,7 +46,24 @@ const Docxtemplater = class Docxtemplater {
 					"The first argument of docxtemplater's constructor must be a valid zip file (jszip v2 or pizzip v3)"
 				);
 			}
-			this.loadZip(zip).compile();
+			this.loadZip(zip);
+			// remove the unsupported modules
+			this.modules = this.modules.filter(module => {
+				if (module.supportedFileTypes) {
+					if (!Array.isArray(module.supportedFileTypes)) {
+						throw new Error(
+							"The supportedFileTypes field of the module must be an array"
+						);
+					}
+					const isSupportedModule = module.supportedFileTypes.includes(this.fileType);
```

Comment:
Can you instead use indexOf !== -1 ?

I tend to avoid using includes since it is ES6 and is not compiled to indexOf in our build